src/EnergyPlus/Material.cc

```diff
@@ -257,7 +257,10 @@ void GetMaterialData(EnergyPlusData &state, bool &ErrorsFound) // set to true if
             MaterNum = ip->getIDFObjNum(state, state.dataHeatBalMgr->CurrentModuleObject, counter);
 
             // Load the material derived type from the input data.
-            auto thisMaterial = state.dataMaterial->Material(MaterNum);
+            delete state.dataMaterial->Material(MaterNum);
```

Comment:
Don't need to do this.

Reply:
right, will not need to delete when the previous new MaterialBase is removed